projects/kit/components/input-phone-international/input-phone-international.style.less

```diff
@@ -35,3 +35,9 @@
     color: var(--tui-text-secondary);
     margin-inline-end: 0.25rem;
 }
+
+.t-search {
+    position: sticky;
+    top: 0.25rem;
+    margin: 0.25rem;
```

Comment:
Please add `background: var(--tui-background-elevation-2)` in case textfield will have transparency.